ush/python/ocean/MLDnc.py

```diff
@@ -87,42 +90,80 @@
 
 ncfiles=xr.open_mfdataset(afiles)
 
-var=ncfiles['mixed_layer_thickness']
-dvar=var-var[0]
-for k in range(var.shape[0]):
+varr=ncfiles['mixed_layer_thickness']
+dvarr=varr-varr[0]
+var_name = 'MLD' 
+units = '(m)'
+delta_var = 10
+delta_dvar = 5
+
+
+for k in range(varr.shape[0]):
+
+   lon = np.asarray(varr[k].Longitude)
+   lat = np.asarray(varr[k].Latitude)
+   lon_geo = np.asarray([ln-360 if ln>=180 else ln for ln in lon])
+   lat_geo = lat
+   var = np.asarray(varr[k])
+   dvar = np.asarray(dvarr[k])
+
    fhr=k*6
    fig=plt.figure(figsize=(15,5))
-   plt.suptitle(storm.upper()+tcid.upper()+'  '+'Ver Hr '+"%3d"%(fhr)+'  (IC='+cycle+'): \n MLD & Change [m]',fontsize=15)
+   plt.suptitle(storm.upper()+tcid.upper()+'  '+'Ver Hr '+"%3d"%(fhr)+'  (IC='+cycle+'): '+var_name + ' & Change '+units,fontsize=15)
+
    ax121=plt.subplot(121)
-   var[k].plot.contourf(levels=np.arange(0,85,5),vmin=0,vmax=90,cmap='RdBu_r')
-   plt.plot(cx,cy,color='gray')
+   kw = dict(levels=np.arange(np.floor(np.nanmin(var)),np.ceil(np.nanmax(var)),delta_var))
```

Comment:
Do we want to implement a "moving" colorbar scale that will change from storm to storm (as is coded here)? Or stick with pre-defined values in the original code?

Reply:
I gave some consideration to this point and decided to propose a adjustable color bar range based on the minimum and maximum values for the specific variable to be plotted. The reason for this is that the range of values for different variables can change considerably depending on the basin. However, having a fixed color range for the different fields does facilitate the comparison of these fields between different storms and basins. So I suggest to give it a try to the adjustable range and then reconsider what option is best.